src/main/java/Duke.java

```diff
@@ -1,10 +1,236 @@
+        File f = new File(filePath); // create a File for the given file path
+        Scanner s = new Scanner(f); // create a Scanner using the File as the source
+        while (s.hasNext()) {
+            String currentLine = s.nextLine();
+            String lineType = currentLine.split("\\|", 3)[0];
+            String lineIsDone = currentLine.split("\\|", 3)[1];
+            String lineAction = currentLine.split("\\|", 3)[2];
+            String lineDescription;
+            String lineDueDate;
+            switch(lineType){
+                case "[T]":
+                    Todo oldTodo = new Todo(lineAction, Boolean.valueOf(lineIsDone));
+                    addTask(oldTodo, true);
+                    break;
+                case "[D]":
+                    lineDescription = lineAction.split("/by", 2)[0];
+                    lineDueDate = lineAction.split("/by", 2)[1];
+                    Deadline oldDeadline = new Deadline(lineDescription, Boolean.valueOf(lineIsDone), lineDueDate);
+                    addTask(oldDeadline, true);
+                    break;
+                case "[E]":
+                    lineDescription = lineAction.split("/at", 2)[0];
+                    lineDueDate = lineAction.split("/at", 2)[1];
+                    Event oldEvent = new Event(lineDescription, Boolean.valueOf(lineIsDone), lineDueDate);
+                    addTask(oldEvent, true);
+                    break;
+            }
+        }
+    }
+
+
+
+    public static void main(String[] args) {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        String greet = "Hello I'm Duke\n"
+                + "What can I do for you?";
+        String exit = "Bye. Hope to see you again soon!";
+
+
+        printBorder();
+        System.out.println("Hello from\n" + logo);
+        System.out.println(greet);
+        printBorder();
+
+        try {
+            printFileContents("data/duke.txt");
+
+        }
+        catch (FileNotFoundException e) {
+            System.out.println("File not found");
+        }
+
+        while (!lineInput.equals("bye")) {
+            printBorder();
+            String[] inputArray = lineInput.split(" ", 2);
+            String inputCommand = inputArray[0];
+            String inputAction = null;
+            if (inputArray.length > 1){
+                inputAction = inputArray[1];
+            }
+
+            switch(inputCommand){
+                case "":
+                    break;
+                case "list":
+                    listTasks();
+                    break;
+                case "done":
+                    try {
+                        markTaskDone(Integer.parseInt(inputAction) - 1);
+                    }
+                    catch (NullPointerException e){

```

Comment:
Coding standard violation: consider using Egyptian style brackets (should have space bar before curly bracket).